Pod/Classes/Highlightr.swift

```diff
@@ -78,50 +78,54 @@ open class Highlightr
         }
         let themeString = try! String.init(contentsOfFile: defTheme)
         theme =  Theme(themeString: themeString)
-
+        
         
         return true
     }
     
-    /**
-     Takes a String and returns a NSAttributedString with the given language highlighted.
-     
-     - parameter code:           Code to highlight
-     - parameter languageName:   Language name or alias
-     - parameter fastRender:     Defaults to true - When *true* will use the custom made html parser rather than Apple's solution.
-     
-     - returns: NSAttributedString with the detected code highlighted.
-     */
-    open func highlight(_ code: String, as languageName: String, fastRender: Bool = true) -> NSAttributedString?
+    /// Takes a String and returns a NSAttributedString with the given language highlighted.
+    ///
+    /// - Parameters:
+    ///   - code: Code to highlight.
+    ///   - languageName: Language name or alias.
+    ///   - needsFastRender: Defaults to true - When *true* will use the custom made html parser rather than Apple's solution.
+    /// - Returns: NSAttributedString with the detected code highlighted.
+    open func highlight(_ code: String, language languageName: String? = nil, needsFastRender: Bool = true) -> NSAttributedString?
```

Comment:
You changed the highlight method signature (from as: to language:), this was renamed recently in an attempt of following the newer Swift 3 style of less verbose methods, and I prefer to keep it this way. Would it be possible to revert this change in your PR?